renderer/pages/oracles/new.js

```diff
@@ -378,20 +393,53 @@ function NewVotingPage() {
 
                 <PresetFormControlInputBox>
                   <DnaInput
-                    id="oracleReward"
-                    value={oracleReward}
-                    min={minOracleReward}
-                    onChange={handleChange}
+                    id="totalVotingFunds"
+                    value={totalVotingFunds}
+                    min={minOracleReward * committeeSize}
+                    onChange={value => {
+                      send('CHANGE', {
+                        id: 'oracleReward',
+                        value: value / committeeSize,
+                      })
+                      send('CHANGE', {
+                        id: 'totalVotingFunds',
+                        value,
+                      })
+                    }}
                   />
                   <PresetFormControlHelperText>
-                    {t('Total oracles rewards: {{amount}}', {
-                      amount: dna(oracleReward * committeeSize),
+                    {t('Min reward per oracle: {{amount}}', {
+                      amount: dna(
+                        (oracleReward * (100 - Math.min(100, ownerFee))) / 100
+                      ),
                       nsSeparator: '!',
                     })}
                   </PresetFormControlHelperText>
                 </PresetFormControlInputBox>
               </PresetFormControl>
 
+              <VotingInlineFormControl
+                htmlFor="ownerFee"
+                label={t('Owner fee')}
+                tooltip={t('% of the Total funds you receive')}
+              >
+                <PercentInput
+                  id="ownerFee"
+                  value={ownerFee}
+                  onChange={handleChange}
+                />
+
+                <PresetFormControlHelperText>
```

Comment:
better use `NewOracleFormHelperText` as `PresetFormControlHelperText` intended to use in `PresetFormControl` component